packages/react/src/useUploadThing.ts

```diff
@@ -1,23 +1,43 @@
-import { useState } from "react";
+import { use, useRef, useState } from "react";
 
-import { DANGEROUS__uploadFiles } from "uploadthing/client";
+import { DANGEROUS__uploadFiles, getUtUrl } from "uploadthing/client";
 import type { ExpandedRouteConfig, FileRouter } from "uploadthing/server";
 
 import { useEvent } from "./utils/useEvent";
 import useFetch from "./utils/useFetch";
 
+const fetchEndpointData = async (endpoint: string) => {
```

Comment:
Am I blind, or where is this actually using the endpoint?

Reply:
oh it isn't - endpoint is filtered in the hook. can remove this param